src/lib.rs

```diff
@@ -59,6 +60,48 @@ pub enum DisplayState {
     Editable,
 }
 
+///Controls the visual output of the `HexView` struct.
+///
+///There are various options which can be altered. For a detailed description of them, please see the fields below.
+///For the changes to apply, you need to use [`set_config`].
+///
+///[`set_config`]: struct.HexView.html#method.set_config
+#[derive(Debug, Clone, Copy)]
+pub struct HexViewConfig {
+    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the `bytes_per_group` value.
+    ///
+    /// Default is 16.
```

Comment:
@palako your PR still isn't right. I mean.. I literally wrote what you have to write in https://github.com/hellow554/cursive_hexview/pull/7#discussion_r552550060 and still you somehow didn't do it.

Am I unclear to you or is it something else? Please correct this one.